PIL/Image.py

```diff
@@ -2531,6 +2543,33 @@ def registered_extensions():
         init()
     return EXTENSION
 
+def register_decoder(name, decoder):
+    """
+    Registers an image decoder.  This function should not be
+    used in application code.
```

Comment:
Scroll up. This is the same as register_extension/save/open/mime/foo.  It's not for application code, it's for plugin developers.